modules/core/src/main/java/org/apache/ignite/spi/discovery/tcp/TcpDiscoverySpi.java

```diff
@@ -291,6 +292,21 @@
     /** Default connection recovery timeout in ms. */
     public static final long DFLT_CONNECTION_RECOVERY_TIMEOUT = IgniteConfiguration.DFLT_FAILURE_DETECTION_TIMEOUT;
 
+    /** @see IgniteSystemProperties#IGNITE_DISCOVERY_CLIENT_RECONNECT_HISTORY_SIZE */
+    public static final int DFLT_DISCOVERY_CLIENT_RECONNECT_HISTORY_SIZE = 512;
+
+    /** @see IgniteSystemProperties#IGNITE_NODE_IDS_HISTORY_SIZE */
+    public static final int DFLT_NODE_IDS_HISTORY_SIZE = 50;
+
+    /** @see IgniteSystemProperties#IGNITE_DISCO_FAILED_CLIENT_RECONNECT_DELAY */
+    public static final int DFLT_DISCO_CLIENT_RECONNECT_DELAY = 10_000;
```

Comment:
I think it is better to use the same name as the prop's name: `DFLT_DISCO_FAILED_CLIENT_RECONNECT_DELAY`

Reply:
Fixed.